tests/scripts/thread-cert/Cert_5_1_06_RemoveRouterId.py

```diff
@@ -100,27 +100,32 @@ def test(self):
         msg = router1_messages.next_coap_message("0.02")
         msg.assertCoapMessageRequestUriPath("/a/as")
 
-        msg = leader_messages.next_coap_message("2.04")
+        leader_messages.next_coap_message("2.04")
 
         # 3 - Router1
-        router1_messages.next_mle_message(mle.CommandType.PARENT_REQUEST)
-        leader_messages.next_mle_message(mle.CommandType.PARENT_RESPONSE)
+        msg = router1_messages.next_mle_message(mle.CommandType.PARENT_REQUEST)
+        msg.assertSentWithHopLimit(255)
+        msg.assertSentToDestinationAddress("ff02::2")
+        msg.assertMleMessageContainsTlv(mle.Mode)
+        msg.assertMleMessageContainsTlv(mle.Challenge)
+        msg.assertMleMessageContainsTlv(mle.ScanMask)
+        msg.assertMleMessageContainsTlv(mle.Version)
+
+        msg = router1_messages.next_mle_message(mle.CommandType.CHILD_ID_REQUEST)
+        msg.assertSentToNode(self.nodes[LEADER])
+        msg.assertMleMessageContainsTlv(mle.Response)
+        msg.assertMleMessageContainsTlv(mle.LinkLayerFrameCounter)
+        msg.assertMleMessageContainsOptionalTlv(mle.MleFrameCounter)
+        msg.assertMleMessageContainsTlv(mle.Mode)
+        msg.assertMleMessageContainsTlv(mle.Timeout)
+        msg.assertMleMessageContainsTlv(mle.Version)
+        msg.assertMleMessageContainsTlv(mle.TlvRequest)
+        msg.assertMleMessageDoesNotContainTlv(mle.AddressRegistration)
 
-        router1_messages.next_mle_message(mle.CommandType.CHILD_ID_REQUEST)
-        msg = leader_messages.next_mle_message(mle.CommandType.CHILD_ID_RESPONSE)
-        msg.assertSentToNode(self.nodes[ROUTER1])
-            
         msg = router1_messages.next_coap_message(code="0.02", uri_path="/a/as")
         msg.assertCoapMessageContainsTlv(network_layer.MacExtendedAddress)
         msg.assertCoapMessageContainsTlv(network_layer.Status)
```

Comment:
add one common `check_address_solicit()` function? like `check_address_release()` in command.py

In this scenario, there should be RLOC16 TLV in address solicit, suggest to use one parameter to indicate whether RLOC16 TLV should be included.